### PR TITLE
Added stubs for oscap tailoring files [6.3 Feature]

### PR DESCRIPTION
**Pull request by ntkathole**

### CODE REVIEW

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 1]
```diff
@@ -0,0 +1,334 @@
+"""Test class for Tailoring Files
+
+:Requirement: tailoringfiles
+
+:CaseAutomation: Automated
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: remove this..

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
+        :steps:
+
+            1. Create a valid scap content
+            2. Upload a Mutually exclusive tailoring file
+            3. Associate the scap content with tailoring file
+
+        :CaseAutomation: notautomated
+
+        :expectedresults: Association should fail
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: Association should give some warning

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+
+        :expectedresults: ARF report should be sent to satellite
+                         reflecting the changes done via tailoring files
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: @tier3

**Comment by oshtaier** [comment]
I would basically increase tier + 1 for all test cases :) however probably `Oscap enabled on capsule and scap tailoring file` doesn't mean that we should prepare something for our tests and these should be provided when satellite is installing. If not then it will be more complex.

**Comment by san7ket** [comment]
yes, here capsule can be an external capsule , so it was mentioned. 
You are right , we are not setting anything in the tests

**Comment by renzon** [comment]
I would start the Review here, but it seems @oshtaier and @jyejare are already doing that ;)

**Comment by sghai** [comment]
@kbidarkar : Hey Please review the stubs once.

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+        :CaseAutomation: notautomated
+
+        :expectedresults: Association should be successful
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: @tier2

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+        :CaseAutomation: notautomated
+
+        :expectedresults: Association should give some warning
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: tier2

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+        :CaseAutomation: notautomated
+
+        :expectedresults: The tailoring file should be downloaded
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: tier2

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 7]
```diff
@@ -0,0 +1,333 @@
...
+        :CaseAutomation: notautomated
+
+        :expectedresults: Get meaningful message about what went wrong
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: tier2

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 8]
```diff
@@ -0,0 +1,333 @@
...
+        :CaseAutomation: notautomated
+
+        :expectedresults: Tailoring file should be deleted
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: tier4

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 9]
```diff
@@ -0,0 +1,333 @@
...
+
+        :expectedresults: ARF report should be sent to satellite reflecting
+                         the changes done via tailoring files
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: tier4

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 10]
```diff
@@ -0,0 +1,333 @@
...
+
+        :expectedresults: ARF report should be sent to satellite
+                         reflecting the changes done via tailoring files
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
```
Comment (san7ket) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: tier4

**Comment by ntkathole** [comment]
@san7ket , @oshtaier...increased tiers +1

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 11]
```diff
@@ -0,0 +1,340 @@
...
+:Upstream: No
+"""
+
+from robottelo.decorators import (
+    run_only_on,
+    tier1,
+    tier2,
+    tier3,
+    tier4,
+    stubbed
```
Comment (sghai) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: Lets put in alphabetical order

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 12]
```diff
@@ -0,0 +1,340 @@
...
+        """ Download the tailoring file from satellite
+
+        :id: 001a75c1-1579-4169-80a5-1a39e8afdd63
+
+        :setup: tailoring file
+
+        :steps:
+
+            1. Upload a tailoring file
+            2. Create tailoring file in satellite
```
Comment (sghai) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: Steps 1,2  can be merged.

**Comment by ntkathole** [comment]
@oshtaier and @sghai one more ACK pending

**Comment by sghai** [comment]
ACK

**Pull request merged**